drivers/flash/nrf_qspi_nor.c

```diff
@@ -476,17 +476,42 @@ static inline int qspi_nor_read_id(struct device *dev,
 	return 0;
 }
 
+#if (CONFIG_HEAP_MEM_POOL_SIZE > 0)
+u8_t *rx_ram_buf;
+u8_t *tx_ram_buf;
```

Comment:
```suggestion
static u8_t *rx_ram_buf = NULL;
static u8_t *tx_ram_buf = NULL;
```

Reply:
AFACT these vars are only used in one function each, why not declare them inside of those functions?